unityfiles/Assets/Scripts/Game/Player/PlayerWeaponController.cs

```diff
@@ -103,6 +103,14 @@ public class PlayerWeaponController : MonoBehaviour {
         this.availableWeapons = tmp;
     }
 
+    void Start(){
+        ChangeWeapon(1);
```

Comment:
Hva er tallet her?

Reply:
QOL bytter automatisk til 1 v\pen ved start saa man ikke gaar med tomme hender i starten av levelen

Reply:
ok. legger du inn en kommentar bak for fremtiden?